bigtop-packages/src/common/logstash/install_logstash.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
```

Comment:
Where does this patch come from and why this is necessary instead of pulling an official released version?